jscomp/super_errors/super_location.ml

```diff
@@ -38,17 +38,16 @@ let print_filename ppf file =
   | real_file -> Format.fprintf ppf "%s" (Location.show_filename real_file)
 
 let print_loc ~normalizedRange ppf loc =
-  setup_colors ();
```

Comment:
cc @jchavarri just in case this affects the ocaml-language-server integration, though I kinda doubt it?

The tldr here is that passing or not passing `-color` works as expected everywhere now, as opposed to not working properly on bsc before (`bsc -color never` wouldn't actually remove colors). bsb isn't affected by this change.

Reply:
Thanks for the heads up @chenglou! For the lang server, no colors (or rather, no ansi escape chars) reach the server, so the output parser doesn't include any processing for them.

The reason is that ninja detects if it's printing into a "smart terminal" and strips these ansi chars if that's not the case (see https://github.com/BuckleScript/bucklescript/issues/2015 for more info).